### DISCUSSION TITLE
how to deal with the problem "No results found on input."

### DISCUSSION THREAD

**Discussion by liyuelan** [antismash/antismash#562] [Q&A] [answered]
I downloaded amount of nucleotides from NCBI genbank and refseq, including .gbff/.fna. And then batch ran them locally using version 6.1 (installed by conda). I found that there were always half of genomes with no BGC prediction, the index.html said "No results found on input." I tried to change the paremeters, to use --genefinding tools prodigal-m. It only worked on a small part of nucleotide. 
I tried to submit it on the website, and gained the same result "No results found on input."
During the analysis, I found a print "No CDS features found in record 'CP021552.1', running gene finding"
Can you tell me, what's wrong with my process?

**Comment by SJShaw** [comment, marked as answer]
Depending on your input, this can be the correct output. Small contigs, contigs with large gaps, etc will all regularly not have any secondary metabolites found.

As for your CP021552.1 example, since it's a 2.5 megabase genome, it's quite likely that it's entirely primary metabolism and won't have any secondary metabolites. Of the Bifidobacterium breve entries in the antiSMASH database, most only have a single secondary metabolite cluster to start with.

If you're really desperate, you can use the loose ruleset with `--hmmdetection-strictness loose`, however it will likely only find some  primary metabolism as false positives.

**Reply by liyuelan** [reply]
> Thank you for your timely response and it is useful to me.
